src/aap_eda/core/tasking/__init__.py

```diff
@@ -40,6 +44,57 @@
     None,
 ]
 
+redis_prefix = f"EDA{service_id()[0:8]}-rq"
```

Comment:
The max size for a key in redis is 512M, even if the chance for a collision is small, do we need to truncate it?